velox/row/tests/UnsafeRowTest.cpp

```diff
@@ -94,6 +94,63 @@ class UnsafeRowFuzzTests : public ::testing::Test {
     }
   }
 
+  void testRoundTrip(const RowVectorPtr& data) {
```

Comment:
Is there copy-paste from the existing code in this file? If so, would it be possible to refactor?

CC: @rui-mo @pedroerp